File: META_Moez/Metastone_web/src/tn/edu/esprit/model/Tournoi.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package tn.edu.esprit.model;

import java.sql.Date;

/**
 *
 * @author proinfo.tn
 */
public class Tournoi {

     private int id_tournoi, Nbr_joueur;
     private Date Date;
     private String Description;
     private String Createur;
     private String Recompence ;
   
       
//constructor
    public Tournoi( Date Date, String Description, String Createur, int Nbr_joueur,  String Recompence) {
        this.Date = Date;
        this.Description = Description;
        this.Createur = Createur;
        this.Nbr_joueur = Nbr_joueur;
        this.Recompence = Recompence;
        
        
    }
    public Tournoi(int id_tournoi, Date Date, String Description, String Createur,int Nbr_joueur, String Recompence) {
        this.id_tournoi = id_tournoi;
        this.Date = Date;
        this.Description = Description;
        this.Createur = Createur;
        this.Nbr_joueur = Nbr_joueur;
        this.Recompence = Recompence;
        
    }

    //..
    public Tournoi ()
    {
       
 
     
        
    }

    public int getId_tournoi() {
        return id_tournoi;
    }

    public void setId_tournoi(int id_tournoi) {
        this.id_tournoi = id_tournoi;
    }

    public int getNbr_joueur() {
        return Nbr_joueur;
    }

    public void setNbr_joueur(int Nbr_joueur) {
        this.Nbr_joueur = Nbr_joueur;
    }

    public Date getDate() {
        return Date;
    }

    public void setDate(Date Date) {
        this.Date = Date;
    }

    public String getDescription() {
        return Description;
    }

    public void setDescription(String Description) {
        this.Description = Description;
    }

    public String getCreateur() {
        return Createur;
    }

    public void setCreateur(String Createur) {
        this.Createur = Createur;
    }

    public String getRecompence() {
        return Recompence;
    }

    public void setRecompence(String Recompence) {
        this.Recompence = Recompence;
    }
    
    
    
    
    
    
    
    
    
    
    
    
    
    
   
    
}

File: META_Moez/Metastone_web/src/Entity/Tournoi.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Tournoi
 *
 * @ORM\Table(name="tournoi")
 * @ORM\Entity
 */
class Tournoi
{
    /**
     * @var int
     *
     * @ORM\Column(name="Id_tournoi", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idTournoi;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="Date", type="date", nullable=false)
     */
    private $date;

    /**
     * @var string
     *
     * @ORM\Column(name="Description", type="string", length=255, nullable=false)
     */
    private $description;

    /**
     * @var string
     *
     * @ORM\Column(name="Createur", type="string", length=255, nullable=false)
     */
    private $createur;

    /**
     * @var int
     *
     * @ORM\Column(name="Nbr_joueur", type="integer", nullable=false)
     */
    private $nbrJoueur;

    /**
     * @var float
     *
     * @ORM\Column(name="Recompence", type="float", precision=10, scale=0, nullable=false)
     */
    private $recompence;

    public function getIdTournoi(): ?int
    {
        return $this->idTournoi;
    }

    public function getDate(): ?\DateTimeInterface
    {
        return $this->date;
    }

    public function setDate(\DateTimeInterface $date): self
    {
        $this->date = $date;

        return $this;
    }

    public function getDescription(): ?string
    {
        return $this->description;
    }

    public function setDescription(string $description): self
    {
        $this->description = $description;

        return $this;
    }

    public function getCreateur(): ?string
    {
        return $this->createur;
    }

    public function setCreateur(string $createur): self
    {
        $this->createur = $createur;

        return $this;
    }

    public function getNbrJoueur(): ?int
    {
        return $this->nbrJoueur;
    }

    public function setNbrJoueur(int $nbrJoueur): self
    {
        $this->nbrJoueur = $nbrJoueur;

        return $this;
    }

    public function getRecompence(): ?float
    {
        return $this->recompence;
    }

    public function setRecompence(float $recompence): self
    {
        $this->recompence = $recompence;

        return $this;
    }


}
